web/src/index.jsx

```diff
@@ -1,11 +1,13 @@
-import * as React from "react";
-import * as ReactDOM from "react-dom";
+import React from "react";
+import { createRoot } from "react-dom/client";
 import ReplicatedErrorBoundary from "./components/shared/ErrorBoundary";
 import Root from "./Root";
 
-ReactDOM.render(
+const container = document.getElementById("app");
+const root = createRoot(container); // createRoot(container!) if you use TypeScript
+
+root.render(
   <ReplicatedErrorBoundary>
     <Root />
-  </ReplicatedErrorBoundary>,
-  document.getElementById("app")
+  </ReplicatedErrorBoundary>
 );
```

Comment:
Following the guidance here. https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html